### ISSUE TITLE
Multiple Encoder Option for Training Base Codec

### ISSUE THREAD

**Issue by kjungwoo03** [EIDOSLAB/Efficient-PIC-with-Variance-Aware-Masking#3]
Thank you for your nice work!

I have a question about training this codec from scratch,

In your WACV'25 paper, the paper mentioned that input image x is encoded into both latent base (y_b) and top (y_t). For that process, mutiple_encoder option should be True in codec.

However, in training recipe you provided in README and also in pretrained codec's arguments, that option is set to be False.

Is that pretrained codec represents the method presented in your paper?

Thank you again for your nice work!

Best regards,
